ci/do_ci.sh

```diff
@@ -85,21 +94,22 @@ if [[ "$1" == "bazel.release" ]]; then
   setup_clang_toolchain
   echo "bazel release build with tests..."
   bazel_binary_build release
+  BAZEL_RELEASE_OPTIONS="test ${BAZEL_TEST_OPTIONS} -c opt ${MEMORY_TEST_EXACT_ARGS}"
 
   if [[ $# -gt 1 ]]; then
     shift
     echo "Testing $* ..."
     # Run only specified tests. Argument can be a single test
     # (e.g. '//test/common/common:assert_test') or a test group (e.g. '//test/common/...')
-    bazel_with_collection test ${BAZEL_TEST_OPTIONS} -c opt $*
+    bazel_with_collection $BAZEL_RELEASE_OPTIONS $*
   else
     echo "Testing..."
     # We have various test binaries in the test directory such as tools, benchmarks, etc. We
     # run a build pass to make sure they compile.
 
     bazel build ${BAZEL_BUILD_OPTIONS} -c opt //include/... //source/... //test/...
```

Comment:
If I read it correctly, ${MEMORY_TEST_EXACT_ARGS} isn't included here? If so this will double CI time for release as the line below will rebuild everything under //test/...

Reply:
great catch; fixed.